core/src/main/java/org/apache/calcite/runtime/CalciteResource.java

```diff
@@ -43,6 +43,9 @@
   @BaseMessage("APPLY operator is not allowed under the current SQL conformance level")
   ExInst<CalciteException> applyNotAllowed();
 
+  @BaseMessage("Accept non path is not allowed under the current SQL conformance level")
```

Comment:
Is it better to change the message like "JSON path expression must be specified after the JSON value expression"? If so the method name should be changed as well.

Reply:
hi @zhztheplayer  Well,Let me modify it .

Reply:
hi @zhztheplayer  I have modified this error description.